### ISSUE TITLE
Allow for disabling redirects

### ISSUE THREAD

**Issue by samtingleff** [digi-trust/dt-cdn#23]
Expose a new option `redirects` within `initialize()`. A value of false will prevent the tag from rewriting links to route a redirect through redirect.html to acquire a first party context.

```
  DigiTrust.initialize({
    member: "IzXoEURCkm",
    site: "ztnV3gXrYQ",
    sample: 1,
    redirects: false
   }
```

**Comment by samtingleff** [comment]
Fixed in 1.2.0

**Comment by samtingleff** [comment]
Note that #25 changes this behavior to be the default (redirects=false).
